### PR TITLE
Use LayoutManager to calculate height of UILabel its height

### PR DESCRIPTION
**Pull request by sergdort**
### Description

We have an issue with [`LabelStylesheet`](https://github.com/Babylonpartners/Bento/issues/82) where it does not respect `numberOfLines` to calculate the height of the label

### Solution 

Use prototype label to calculate its height

### CODE REVIEW

**Comment by andersio** [comment]
If we move to use prototype cell for sizing, we should make everything be computed consistently. The implementations should be moved to `TextBoundComputing`, and there is no point to have two overloads for `String` and `NSString`.

**Comment by sergdort** [comment]
@andersio I think we can't move implementation to `TextBoundComputing` because in this case it uses `UILabel` prototype, in another it may use something else e.g UITextView

**Comment by sergdort** [comment]
@andersio Addressed your comments

File: StyleSheets/StyleSheets/LabelStyleSheet.swift [hunk 1]
```diff
@@ -15,20 +14,20 @@ open class LabelStyleSheet: ViewStyleSheet<UILabel>, TextBoundComputing {
...
+
     public init(
         backgroundColor: UIColor? = .clear,
         font: UIFont = UIFont.preferredFont(forTextStyle: .body),
         textColor: UIColor = .black,
         textAlignment: TextAlignment = .leading,
         numberOfLines: Int = 0,
-        lineBrealMode: NSLineBreakMode = .byTruncatingTail
-        ) {
+        lineBreakMode: NSLineBreakMode = .byTruncatingTail
```
Comment (diegopetrucci) [inline StyleSheets/StyleSheets/LabelStyleSheet.swift]: 👌

**Pull request merged**